packages/snap/src/permissions.ts

```diff
@@ -59,6 +61,7 @@ export const originPermissions = new Map<string, string[]>([
       KeyringRpcMethod.RejectRequest,
       // Custom methods
       InternalMethod.SetConfig,
+      InternalMethod.GetConfigs,
```

Comment:
We will need to add the paymaster internal methods here too (in subsequent PR?)

Reply:
Yes this is in a subsequent pr to add deployment of the paymaster from the UI